Movement.cpp

```diff
@@ -0,0 +1,202 @@
+/* Roborregos 2020.
+ * This Movement class has all the functions
+ * To move the robot to anywhere on the map.
+ * This class works with all robot enginees.
+ * To get more information, go to Movement.h file.
+*/
+#include "Movement.h"
+
+Movement::Movement() {}
+
+void Movement::advance(const double desire, const double desireUltrasonic) {
+  double encoder1 = 0;
+  double encoder2 = 0;
```

Comment:
The encoders do not work like this. You do not need these variables in here. Check:
https://github.com/tomvik/RoBorregos-Japon/blob/master/libraries/Movimiento/Movimiento.h (Look for eCount1)
And https://github.com/tomvik/RoBorregos-Japon/blob/master/libraries/Movimiento/Movimiento.cpp (Look for line 685 and 701)